README.md

```diff
@@ -25,51 +25,47 @@ A design system and [React](https://facebook.github.io/react/) component library
 
 Mineral UI is distributed as a multitude of [npm packages](https://www.npmjs.com/org/mineral-ui). E.g., to install the Button package:
 
-1. Install project [peer dependencies](./docs/peer-dependencies.md)
+1. Install the Mineral UI packages that you wish to use
 
-  ```sh
-  npm install --save react react-dom glamorous glamor
-  ```
-
-2. Install the Mineral UI packages that you wish to use
-
-  ```sh
-  npm install --save @mineral-ui/button @mineral-ui/component-utils
-  ```
+```sh
+npm install --save @mineral-ui/button @mineral-ui/component-utils
+```
 
+2. Install any missing peer dependencies reported by `npm` or `yarn`
 3. Then, in your app, import and use just like any other React component
 
-  _ES2015_
```

Comment:
The indentation was there because then it was obvious that the code block belonged with the preceeding number in the list of steps. Not a big deal to change it, though.